### PR TITLE
Created partial for collaborators

### PR DESCRIPTION
**Pull request by AbdullahAlger**

### CODE REVIEW

**Comment by AbdullahAlger** [comment]
Haml is really nice once you get used to it.

File: app/views/wikis/index.html.haml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            %small
+              Created by:
+              = wiki.user.name
+              = time_ago_in_words(wiki.created_at)
+              ago
+    .col-xs-1
+    .col-xs-4
+      .new-wiki-button
+        = link_to "Create wiki", new_wiki_path, class: "btn btn-success"
+      - if current_user.standard? || current_user.admin?
```
Comment (eliotsykes) [inline app/views/wikis/index.html.haml]: What would be a good method to write to encapsulate and describe the logic on line 19 here?

Comment (AbdullahAlger) [inline app/views/wikis/index.html.haml]: I am going to take this out and use only `.standard?`. I just wanted admin to see the button for testing it.

Comment (eliotsykes) [inline app/views/wikis/index.html.haml]: That sounds OK. Naturally choose the option you prefer after this next little thought experiment:

Compare these 2 possibilities:

```haml
- if current_user.standard?
  show upgrade button here
```


```haml
- if current_user.upgradeable?
  show upgrade button here
```
Notice how the `upgradeable?` option documents why this `if` condition is here. The `if` condition is here to show something thats only visible to users who can upgrade.

Compare this to the `current_user.standard?` condition. This does not tell us the real goal of the `if` block. The true goal of the `if` block is not to show something to standard users, it is to show something to users who can upgrade.

For this particular situation, that refactoring might be overkill, but I'd like you to try to think about future conditionals you write and try to come up with methods that reveal the underlying intent. (Also always feel free to add your own methods to the models, or to any class for that matter).

**Pull request merged**